Remove the width on the cards container

renestalder opened the pull request:
<!-- Thanks for contributing to Peregrine CMS! -->

**What does this implement/fix? Explain your changes.**

Ensure the cards on the same line, vertically left and right, as other
components, like the media component.

![Screenshot from 2020-06-22 11-48-21](https://user-images.githubusercontent.com/105358/85273768-54372300-b47e-11ea-92bc-dcb4b6f086d7.png)

**Does this close any currently open issues?**

Known issue reported by @reusr1, not formally in a ticket.

**Any other comments?**

–

**Where has this been tested?**

*Browser (version):* Firefox 77

Commit:
Remove the width on the cards container

Ensure the cards on the same line, vertically left and right, as other
components, like the media component

reusr1 commented:
@renestalder the spacing for the cards does not look right in the mobile breakpoint compared to a `media image` component or the navigation

renestalder commented:
> the spacing for the cards does not look right in the mobile breakpoint compared to a `media image` component or the navigation

Thanks for testing. I will check that.

Commit:
Limit card negative margins to medium breakpoint

Fix an issue where the cards wouldn't be vertically aligned
with other elements

renestalder commented:
@reusr1 Should now look better. I limited the negative margins required to align the cards vertically to only be used in the medium breakpoint upwards.
